scripts/push_annotations_from_files.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Push annotations from JSON files to Synapse entities"""
+
+import synapseclient
+import json
+
+def push_annotations_from_file(file_path, entity_id):
+    """Push annotations from a JSON file to a Synapse entity"""
+    
+    print(f"Pushing annotations from {file_path} to entity {entity_id}")
```

Comment:
A whole other ticket you can create, but experiment with the logger function in Python. https://www.dash0.com/guides/logging-in-python